src/main/java/com/example/earthtalk/domain/user/userDto/UserInfoDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.earthtalk.domain.user.userDto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public record UserInfoDto (
+    Long userId,
+    String nickname,
+    String profile,
+    String introduction,
+    Long totalFollowers,
+    Long totalFollowees,
+    Long wins,
+    Long draws,
+    Long losses
+) {}
```

Comment:
response 내용을 해당 record에서 정의하고 사용하는 것이 좋을 것 같습니다
```suggestion
) {
public static UserInfoResponse from(User user, Long totalFollowers, Long totalFollowees) {
        return new UserInfoResponse(
                user.getId(),
                user.getNickname(),
                user.getProfile(),
                user.getIntroduction(),
                totalFollowers,
                totalFollowees,
                user.getWinNumber(),
                user.getDrawNumber(),
                user.getDefeatNumber()
        );
    }
}
```